src/TimePicker/TimePicker.js

```diff
@@ -183,6 +144,27 @@ module.exports = kind(
 	},
 
 	/**
+	* Sets up meridiems for en-US.
+	* @private
+	*/
+	setupMeridiems: function () {
+		this._strAm = 'AM';
+		this._strPm = 'PM';
+
+		if (this.is24HrMode == null) {
```

Comment:
Is this block still necessary, now that we have set the default to `false` and `null` is no longer a valid value?